AES.Web/Controllers/JobOpeningsController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using AES.Web.OpeningService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using AES.Web.Models;
+using AES.OpeningsSvc;
+
+namespace AES.Web.Controllers
+{
+    public class JobOpeningsController : Controller
+    {
+        // How to set up a cookie 
+
+        // GET: JobOpenings
+        public ActionResult Index()
+        {
+            // Contract being returned
+            //IOpeningSvc JobOpenings = new OpeningService.OpeningSvcClient();
```

Comment:
This should be the right way in the commit. Add a unit test to test this page (there are ones already present showing how to do it) so that when committed, AppHarbor will show if there are any issues.